examples/README.md

```diff
@@ -16,5 +16,6 @@ This folder contains different examples of AMQ Streams custom resources and demo
     * Prometheus Alert
     * Sample Prometheus installation files
     * Sample Grafana installation files
+    * JMX Trans deployment
```

Comment:
JMX Trans is not something we have in AMQ Streams, you should remove it.

Reply:
Removed